contracts/Registry.sol

```diff
@@ -148,9 +191,17 @@ contract Registry {
         // Cannot exit during ongoing challenge
         require(listing.challengeID == 0 || challenges[listing.challengeID].resolved);
 
-        // Remove listingHash & return tokens
+        // Make sure the exit was initialized
+        require(listing.exitTime > 0);
+
+        // Get the time when the exit is no longer valid
+        uint timeExpired = listing.exitTime.add(parameterizer.get("exitTimeExpiry"));
```

Comment:
Does it matter if the `exitTimeExpiry` parameter changes between the call to `initExit` and `finalizeExit`? Would it make sense to calculate the expiry time in `initExit`?

Reply:
To elaborate, if I call `initExit` and `exitTimeExpiry` is set to 7 days, I would think I have that much time to call `finalizeExit` once the waiting period is over. Suppose that during the waiting period, `exitTimeExpiry` changes to 2 days: should I have 2 days to finalize or 7 days?

1) In the case where the exit period end is snapshotted, I have 7 days, as I expect.
2) In the case where it isn't, I may show up 3 days in thinking that I still have time, only to find out that my exit period expired without me realizing it. That sounds pretty annoying.

There may be some case for (2), aside from less storage, but I can't think of any others off the top of my head. Maybe it would prevent some sort of bad behavior?